src/app/components/wrapper-components/GameWrapper.ts

```diff
@@ -51,39 +52,51 @@ export abstract class GameWrapper<P extends Comparable> {
 
     public Player: typeof Player = Player;
 
-    public constructor(protected readonly actRoute: ActivatedRoute,
+    public constructor(activatedRoute: ActivatedRoute,
                        protected readonly connectedUserService: ConnectedUserService,
                        protected readonly router: Router,
                        protected readonly messageDisplayer: MessageDisplayer)
     {
+        super(activatedRoute);
     }
-    public getMatchingComponent(gameName: string): MGPOptional<Type<AbstractGameComponent>> {
+
+    private getMatchingComponent(gameName: string): MGPOptional<Type<AbstractGameComponent>> {
         const gameInfo: MGPOptional<GameInfo> =
             MGPOptional.ofNullable(GameInfo.ALL_GAMES().find((gameInfo: GameInfo) => gameInfo.urlName === gameName));
         return gameInfo.map((gameInfo: GameInfo) => gameInfo.component);
     }
-    protected async afterViewInit(): Promise<boolean> {
-        const gameCreatedSuccessfully: boolean = await this.createGameComponent();
-        if (gameCreatedSuccessfully) {
-            this.gameComponent.node = this.gameComponent.rules.getInitialNode();
+
+    protected async createMatchingGameComponent(): Promise<boolean> {
+        // Method to be called only after view init
```

Comment:
Idéalement, placer le commentaire avant la méthode pour dire ce qu'elle fait. Par exemple ici "This method is to be called only after view init. It will create the game component and initialize its node."

Reply:
m'est avis que notre politique de commentaire est pas trop idéale en vrai, on a tendance à commenter des petits détails intra-fonctions, mais on documente PAS nos fonctions, ce qui est plus important ! À y penser petit à petit